empiric-package/empiric/core/entry.py

```diff
@@ -1,21 +1,22 @@
 from collections import namedtuple
+from typing import Any, List
 
 from empiric.core.utils import str_to_felt
 
 Entry = namedtuple("Entry", ["key", "value", "timestamp", "source", "publisher"])
 
 
-def serialize_entry(entry):
+def serialize_entry(entry) -> List[Any]:
```

Comment:
would like to get the expected types here

Reply:
A list of ints. Technically could try to use a tuple here (because it is immutable & fixed length) but not sure if starknet.py will accept that